fillers/eips/eip3860.py

```diff
@@ -552,6 +579,15 @@ def generate_create_opcode_initcode_test_cases(
                 len(initcode.assemble())
             )
 
+        # Call returns 1 as valid initcode length s[0]==1 && s[1]==1
+        post[to_address(0x200)] = Account(
+            nonce=1,
+            storage={
+                0: 1,
+                1: 1,
```

Comment:
```suggestion
                0: 0,
                1: 1,
```
Change due to the first suggestion.